SharedContent/cpp-winui-template/WinUI3TemplateCpp/App.xaml.h

```diff
@@ -0,0 +1,21 @@
+﻿#pragma once
+
+#pragma push_macro("GetCurrentTime")
+#undef GetCurrentTime
+
+#include "App.xaml.g.h"
+
+#pragma pop_macro("GetCurrentTime")
+
+namespace winrt::WinUI3TemplateCpp::implementation
+{
+    struct App : AppT<App>
+    {
+        App();
+
+        void OnLaunched(Microsoft::UI::Xaml::LaunchActivatedEventArgs const&);
+        
+    private:
+        winrt::Microsoft::UI::Xaml::Window window{ nullptr };
+    };
+}
```

Comment:
For the upcoming 1.x releases, probably ~10 features (which may have multiple samples per feature). I think "Templates" would be a better name, since not all samples will use these templates (for example, we also will have Win32/WPF/WinForms/Console app samples). 

Having to edit all WinUI samples whenever updating the template may be a nuisance, but we might need to do this for early releases of WinAppSDK